### ISSUE TITLE
Different result compared with sklearn.metric.mutual_info_score

### ISSUE THREAD

**Issue by kaixiongg** [Craigacp/MIToolbox#9]
Readme provides the result as 0.02, is there any normalization or what influence the result?
>>> x = np.array([1,1,1,0,0])
>>> y=np.array([1,0,1,1,0])
>>> mutual_info_score(x,y)
0.013844293808390418

**Comment by Craigacp** [comment]
MIToolbox uses log base 2 for all computations, scikit-learn uses log base e. So our units are "bits" and theirs are "nats".

**Comment by kaixiongg** [comment]
Understood. thanks for your reply
